.github/workflows/playwright-ci.yml

```diff
@@ -1,11 +1,11 @@
 name: Playwright tests
 on:
   pull_request:
-    branches: ["main"]
+    branches: ["main", "1.0.0"]
```

Comment:
Why not running the tests on all branches instead?

Reply:
You are right!

Reply:
We should!